dev/packages/beats/aerospike-0.0.1/dataset/namespace/fields/fields.yml

```diff
@@ -1,158 +1,158 @@
 - name: aerospike.namespace
   type: group
-  description: >
+  description: |
     namespace
   release: ga
   fields:
-    - name: client
+  - name: client
+    type: group
+    description: |
+      Client stats.
+    fields:
+    - name: delete
       type: group
-      description: >
-        Client stats.
+      description: |
```

Comment:
Not related directly to this PR but something we should discuss for the future: We have these group descriptions for historical reasons but not 100% sure if these still will make sense in the future. Especially as these are for docs only and were always a challenge to visualise.

Reply:
I didn't know about this. Any opinion on this @exekias ?

Reply:
I'm definitely ok with removing something we don't really show in the docs (or somewhere else)

Reply:
I've updated the PR, removed all descriptions as you suggested.